query-engine/driver-adapters/Cargo.toml

```diff
@@ -22,9 +21,21 @@ bigdecimal = "0.3.0"
 chrono = "0.4.20"
 futures = "0.3"
 
-napi.workspace = true
-napi-derive.workspace = true
-
 [dev-dependencies]
 expect-test = "1"
 tokio.workspace = true
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+napi.workspace = true
+napi-derive.workspace = true
+quaint.workspace = true
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+quaint = { path = "../../quaint" }
+js-sys.workspace = true
+serde-wasm-bindgen.workspace = true
+wasm-bindgen.workspace = true
+wasm-bindgen-futures.workspace = true
+tsify.workspace = true
+ducktor = "0.1.0"
```

Comment:
This dependency, `ducktor`, looks scary tbh. Completely unknown, a few months old with barely any tests. How much work does it remove from our plate?

Reply:
`ducktor` was one of the shortcuts taken (I already had some code using it laying around).

Removed in https://github.com/prisma/prisma-engines/pull/4456/commits/d6d09d6f47999af237eac47771d69db68d00e2ed.

Reply:
Looks like this change broke something:

```
Error: `unwrap_throw` failed
    at __wbindgen_throw (file:///Users/jkomyno/work/prisma/prisma-engines/query-engine/query-engine-wasm/pkg/query_engine_bg.js:977:11)
    at wasm://wasm/01063c0e:wasm-function[5374]:0x352371
    at wasm://wasm/01063c0e:wasm-function[3153]:0x32513b
    at wasm://wasm/01063c0e:wasm-function[93]:0x3902b
```

I'll take another look tomorrow.